src/components/blocks/software/Code/MultilineCodeContent.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Light as SyntaxHighlighter } from 'react-syntax-highlighter';
+
+export const MultilineCodeContent = ({
+  signedIn,
+  dataContainsKey,
+  contentWithObfuscatedKey,
+  contentWithKey,
+  content,
+  displayLanguage,
+}: {
+  signedIn: boolean;
+  dataContainsKey: boolean;
+  contentWithObfuscatedKey: string;
+  contentWithKey: string;
+  content: string;
+  displayLanguage: string;
+}) => {
+  const contentMaybeWithKey = dataContainsKey
```

Comment:
Refactor into a method and maybe the first one could be an non linline if statement. Multiple nested inlines are a bit much and hurt my head!